canonical-websites/www.ubuntu.com
# Flatten legal contributor section

## Done

- Flattened legal contributor section
- Updated navigation
- Updated redirects
- Once live, we will need to update formassembly to use the thank-you page directly; however, redirects will work for now.

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- look at the files in the /legal directory and compare to this [spreadsheet](https://docs.google.com/spreadsheets/d/12kuuGgwRqN3k7wlRc4QKIuZifjvxbynizO_LEjKjBq4/edit#gid=0)
* test the link [/legal/contributors-agreement-faq](http://0.0.0.0:8001/legal/contributors-agreement-faq) and [the old link](http://0.0.0.0:8001/legal/contributors/licence-agreement-faq) redirects
* test the link [/legal/contributors-agreement](http://0.0.0.0:8001/legal/contributors-agreement) and [the old link](http://0.0.0.0:8001/legal/contributors/submit) redirects
* test the link [/legal/contributors-agreement/thank-you](http://0.0.0.0:8001/legal/contributors-agreement/thank-you) and [the old link](http://0.0.0.0:8001/legal/contributors/thank-you) redirects


## Issue / Card

Fixes #4192
